docs/dev.rst

```diff
@@ -71,7 +71,7 @@ Setup quickstart
 
    That will build the app Docker image required for development.
 
-5. Initialize Postgres, Elasticsearch, S3, and SQS.
+5. Initialize Postgres, Elasticsearch, Pub/Sub, S3, and SQS.
```

Comment:
During the reviews so far, I was figuring that while we're starting a period of transition to GCP things, these services still require AWS things, so we could leave the docs as is for now. As part of cleaning up after the transition, we'd go through and overhaul the docs replacing all the AWS things with GCP things.

But updating the docs as we go is cool, too. :+1: